webview/src/plots/components/comparisonTable/cell/ComparisonTableMultiCell.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useCallback } from 'react'
+import cx from 'classnames'
+import { useDispatch, useSelector } from 'react-redux'
+import { ComparisonPlot } from 'dvc/src/plots/webview/contract'
+import { ComparisonTableLoadingCell } from './ComparisonTableLoadingCell'
+import { ComparisonTableMissingCell } from './ComparisonTableMissingCell'
+import styles from '../styles.module.scss'
+import {
+  changeDisabledDragIds,
+  setMultiPlotValue
+} from '../comparisonTableSlice'
+import { PlotsState } from '../../../store'
+import { zoomPlot } from '../../../util/messages'
+
+const getCurrentStep = (stateStep: number | undefined, imgsLength: number) => {
+  if (!stateStep) {
+    return 0
+  }
+  return stateStep > imgsLength - 1 ? imgsLength - 1 : stateStep
```

Comment:
>  This behaviour isn't very intuitive. I had no idea what was happening when running a new experiment. I would expect the image to appear as missing if the image for that step does not exist yet. This will probably simplify the code as well

I chose the new behavior since I was avoiding our sliders having mutatable max length. Example:

https://github.com/iterative/vscode-dvc/assets/43496356/32255c68-74dd-4297-b643-d98cb4b59a40

But if you think the behavior above is preferred though, I'm happy to change it. We also could look into finding the max length across all the plots in the backend so the max length doesn't get altered as much (probably in a followup).

Reply:
I think the slider should have the max number of available steps for all experiments but also show missing if that step is not available for that experiment.

Reply:
> I think the slider should have the max number of available steps for all experiments but also show missing if that step is not available for that experiment.

Ok, so just making sure I'm understanding things correctly. Let's say we have two experiments, one with 12 images, and one with 5 images. We want the plots to render two plots, each with a slider with a range of 0-11?

Reply:
> Maybe we are overcomplicating and shouldn't be syncing these sliders across experiments. It sounded nice, but it might occasionally be frustrating if the user actually wants to compare different steps. We could always further optimize later. Maybe @daavoo or others have ideas here.

I am ok with not syncing sliders

Reply:
Makes sense! I can remove the syncing logic for now and we can try out different ideas in a followup.

Reply:
Sorry for the flip flopping here @julieg18!